### PR TITLE
Reset passwd validation

### PR DESCRIPTION
**Pull request by msaianand**

### CODE REVIEW

**Comment by msaianand** [comment]
[Define and set password rules on UI](https://trello.com/c/JY3k7EIB/62-define-and-set-password-rules-on-ui)

**Comment by msaianand** [comment]
[Define and set password rules on UI](https://trello.com/c/JY3k7EIB/62-define-and-set-password-rules-on-ui)

File: src/main/react-front-end/src/views/Login/CreateAccountComponent.js [hunk 1]
```diff
@@ -127,10 +127,21 @@ export default class CreateAccountComponent extends Component {
...
-      state.passwordError = "Password Doesn't Match";
-      self.setState({ state });
+    if(!(/[a-z]/.test(password))){
+			eventEmitter.emit("errorOccured", "Password must have atleast one lower character");
+		}
+		else if(!(/[A-Z]/.test(password))){
+			eventEmitter.emit("errorOccured", "Password must have atleast one upper character");
+		}
+		else if(!(/[0-9]/.test(password))){
+			eventEmitter.emit("errorOgccured", "Password must have atleast one digit");
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/Login/CreateAccountComponent.js]: Spelling error.

**Review by LinusCastelino — CHANGES_REQUESTED** [review]
Since the same code is used in both files, please create it as a function in constants.js and reference the method for checking the validity of the password.

File: src/main/react-front-end/src/views/Login/CreateAccountComponent.js (hunk 1 above)
Comment (msaianand) [reply src/main/react-front-end/src/views/Login/CreateAccountComponent.js]: corrected spelling error and moved the function to constant.js

**Review by LinusCastelino — COMMENTED** [review]
Logic and approach looks good now.
Need to add additional functionality to this. Will discuss when we meet next

**Comment by aashish-jain** [comment]
Fixed in #281. Closing the PR.

**Pull request closed without merging**